docs-next/pages/guides/configuring-relationships.mdx

```diff
@@ -0,0 +1,243 @@
+We can configure a one-sided version of this:
+
+```typescript
+import { config, createSchema, list } from '@keystone-next/keystone/schema';
+import { text, relationship } from '@keystone-next/fields';
+
+export default config({
+  lists: createSchema({
+    User: list({
+      fields: {
+        name: text()
+      }
+    }),
+    Post: list({
+      fields: {
+        title: text(),
+        content: text(),
+        author: relationship({ ref: 'User', many: false }),
+      }
+    })
+  }),
+});
+```
+
+or a two-sided version:
+
+```typescript
+export default config({
+  lists: createSchema({
+    User: list({
+      fields: {
+        name: text(),
+        posts: relationship({ref: 'Post.author', many: true }),
+      }
+    }),
+    Post: list({
+      fields: {
+        title: text(),
+        content: text(),
+        author: relationship({ ref: 'User.posts', many: false }),
+      }
+    })
+  }),
+});
+```
+
+Note that we have used `many: false` in the `author` field and `many: true` in the `posts` field.
+
+### Many-to-many
+
+If we want a blog where each post can have **many** authors, and each user can be the author of **many** posts, then we have a `many-to-many` relationship.
+We can configure a one-sided version of this:
+
+```typescript
+export default config({
+  lists: createSchema({
+    User: list({
+      fields: {
+        name: text(),
+      }
+    }),
+    Post: list({
+      fields: {
+        title: text(),
+        content: text(),
+        authors: relationship({ ref: 'User', many: true }),
+      }
+    })
+  }),
+});
+```
+
+or a two-sided version:
+
+```typescript
+export default config({
+  lists: createSchema({
+    User: list({
+      fields: {
+        name: text(),
+        posts: relationship({ ref: 'Post.authors', many: true }),
+      }
+    }),
+    Post: list({
+      fields: {
+        title: text(),
+        content: text(),
+        authors: relationship({ ref: 'User.posts', many: true }),
+      }
+    })
+  }),
+});
+```
+
+Note that we have used `many: true` in both the `authors` and `posts` fields.
+
+### One-to-one
+
+If we want a blog where each post has exactly **one** author, and each user is restricted to writing exactly **one** post, then we have a `one-to-one` relationship.
+In this case we can only specify this with a two-sided relationship:
+
+```typescript
+export default config({
+  lists: createSchema({
+    User: list({
+      fields: {
+        name: text(),
+        post: relationship({ ref: 'Post.author', many: false }),
+      }
+    }),
+    Post: list({
+      fields: {
+        title: text(),
+        content: text(),
+        author: relationship({ ref: 'User.post', many: false }),
+      }
+    })
+  }),
+});
+```
+
+Note that we have used `many: false` in both the `authors` and `posts` fields.
+
+## Summary
+
+When configuring a relationship in Keystone there are two key questions you need to answer:
+
+- Do I want a one-sided or two-sided relationship?
```

Comment:
```suggestion
- Do I want to access the relationship from only one-side or both?
```

Or something similar; just feel we should be emphasising this is purely around access/API.

Reply:
This language is specifically goal oriented, not technically oriented, around what they want to be able to do, which will lead them to the technical decision of one-sided or two-sided.

Reply:
Oh wait, I'm reading this backwards, and it's my prose that's very wrong. yes, good suggestion indeed!